README.md

```diff
@@ -1,16 +1,26 @@
-# ARTwarp 
-
-ARTwarp is a MATLAB-based programme for the automated categorisation of tonal
-animal sounds. It has been tested successfully for bottlenose dolphin whistles
-and killer whale calls, but will be applicable to any sound that can be
-described by frequency contours.
-
-These `ARTwarp.m` files have been run successfully using Matlab 2016a and 2016b
-
-`ARTwarp.m` will open whistle contours extracted using Beluga
-
-`ARTwarp_txt.m` provides the option to open `.ctr` (Beluga), `.csv`, or `.txt` files
-
-This version of `ARTwarp_txt` displays the iteration number and number of whistles 
-analysed in the command window while ARTwarp is running. It saves the results at 
-the end of each iteration.
+# ARTwarp 

```

Comment:
This looks strange - it should not show all differences in lines 1-16 which are untouched. Which editor and which operating system do you use? How do you make commits?